src/FirePokemon.java

```diff
@@ -0,0 +1,41 @@
+package src;
+
+public class FirePokemon extends Pokémon {
+    private int flameThrower;
+    private String fireLash;
+
+    public FirePokemon(String sound, String food, int flameThrower, String fireLash) {
+        super(sound, food);
+        this.flameThrower = flameThrower;
+        this.fireLash = fireLash;
+    }
+
+    public int getFlameThrower() {
+        return flameThrower;
+    }
+
+    public String getFireLash() {
+        return fireLash;
+    }
+
+    public void setFlameThrower(int flameThrower) {
+        this.flameThrower = flameThrower;
+    }
+
+    public void setFireLash(String fireLash) {
+        this.fireLash = fireLash;
+    }
+
+    public String nextLevel() {
+        return " level 2!";
+    }
+
+    public void printFoodAndSound() {
```

Comment:
Een methode als "doXandY" is vaak geen goed idee. De SOLID principes beschrijven o.a. SRP (Single Responsibility Principle) wat zegt dat een methode of klasse 1 verantwoordelijkheid moet hebben en je dus geen sound EN food zou moeten printen in dezelfde methode.
Wel kun je iets als een `getBasicInfo` methode maken waarin je alle basis info van de pokemon print, zoals sound en food, height, age, etc.